src/main/scala/fs/fusejna.scala

```diff
@@ -0,0 +1,114 @@
+  def lock(path: String, info: FileInfo, command: FlockCommand, flock: FlockWrapper): Int      = underlying.lock(path, info, command, flock)
+  def mkdir(path: String, mode: ModeInfo): Int                                                 = underlying.mkdir(path, mode)
+  def mknod(path: String, mode: ModeInfo, dev: Long): Int                                      = underlying.mknod(path, mode, dev)
+  def open(path: String, info: FileInfo): Int                                                  = underlying.open(path, info)
+  def opendir(path: String, info: FileInfo): Int                                               = underlying.opendir(path, info)
+  def read(path: String, buffer: Buf, size: Long, offset: Long, info: FileInfo): Int           = underlying.read(path, buffer, size, offset, info)
+  def readdir(path: String, filler: DirectoryFiller): Int                                      = underlying.readdir(path, filler)
+  def readlink(path: String, buffer: Buf, size: Long): Int                                     = underlying.readlink(path, buffer, size)
+  def release(path: String, info: FileInfo): Int                                               = underlying.release(path, info)
+  def releasedir(path: String, info: FileInfo): Int                                            = underlying.releasedir(path, info)
+  def removexattr(path: String, xattr: String): Int                                            = underlying.removexattr(path, xattr)
+  def rename(path: String, newName: String): Int                                               = underlying.rename(path, newName)
+  def rmdir(path: String): Int                                                                 = underlying.rmdir(path)
+  def setxattr(path: String, xattr: String, value: Buf, size: Long, flags: Int, pos: Int): Int = underlying.setxattr(path, xattr, value, size, flags, pos)
+  def statfs(path: String, wrapper: StatvfsWrapper): Int                                       = underlying.statfs(path, wrapper)
+  def symlink(path: String, target: String): Int                                               = underlying.symlink(path, target)
+  def truncate(path: String, offset: Long): Int                                                = underlying.truncate(path, offset)
+  def unlink(path: String): Int                                                                = underlying.unlink(path)
+  def utimens(path: String, wrapper: TimeBufferWrapper): Int                                   = underlying.utimens(path, wrapper)
+  def write(path: String, buf: Buf, bufSize: Long, writeOffset: Long, info: FileInfo): Int     = underlying.write(path, buf, bufSize, writeOffset, info)
+}
+
+/** Our implementation of the DirFiller interface.
+ */
+final class DirFiller(initial: jList[String]) extends DirectoryFiller {
+  def this() = this(jList())
+
+  private var count = 0
+  private val buf = Vector.newBuilder[Path]
+  private def add(s: String): Unit = { buf += path(s) ; count += 1 }
+
+  locally {
```

Comment:
It's old, it's in predef. It's a no-op. It's to make constructor code stand out more.